word-search.py

```diff
@@ -80,9 +81,58 @@ def print_grid(grid):
     # Bottom border
     print(" " * left_padding + "└" + "───┴" * (cols - 1) + "───┘")
 
-def loadWordLibrary():
-    # read words from file and return as list for use in program
-    return
+def loadWordLibrary(grid, min=3, filename= "word-list.txt"):
```

Comment:
Adjust the function name to load_word_library to follow our naming convention

Reply:
updated, thanks!